client/package.json

```diff
@@ -9,15 +9,13 @@
     "virtual-dom": "latest"
   },
   "scripts": {
-    "start": "watchify main.js -o out.js",
-    "build": "browserify main.js -o out.js"
+    "start": "webpack --watch",
+    "build": "webpack"
   },
   "devDependencies": {
-    "browserify": "latest",
-    "babelify": "latest",
-    "babel-plugin-transform-react-jsx": "latest"
-  },
-  "browserify": {
-   "transform": ["babelify"]
+    "babel-core": "^6.26.0",
```

Comment:
`babel-loader` lists `babel-core` as its peerDependency, so we need to explicitly depend on it.